### DISCUSSION TITLE
Progress bar per each budget

### DISCUSSION THREAD

**Discussion by kopach** [firefly-iii/firefly-iii#4346] [General]
**Description**
Hi. I'm just starting to use FireFly III, and it looks/works awesome. Thank you. I do have small feature request which could improve user experience. Is there any chance to add progress bar per each of created budgets on budgets page, so it's easier to quickly understand what is the budget status?

**Solution**
See example mockup:
<img width="407" alt="image" src="https://user-images.githubusercontent.com/8749624/106807787-da992180-6669-11eb-80fa-c1c7badbf75f.png">

**What are alternatives?**
There is Total budget on /budgets page and stacked bar graph on dashboard but that is not that much convenient as viewing budgets specifically on /budgets page.

**Comment by JC5** [comment]
Yes, this will be something that I plan to add in the new layout 👍

**Reply by kopach** [reply]
Oh, that would be awesome, thanks. And do you have maybe already any estimates when new layout will be released?
